src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -1037,7 +1036,9 @@ ArrayList generalResolutionExceptions
 
                                 if (conflictVictims.Length == 0)
                                 {
-                                    foundAtLeastOneUnresolvableConflict = true;
+                                    // This warning is logged regardless of AutoUnify since it means a conflict existed where the reference
+                                    // chosen was not the conflict victor in a version comparison, in other words it was older.
+                                    Log.LogWarningWithCodeFromResources("ResolveAssemblyReference.FoundConflicts", idealRemappingPartialAssemblyName.Name);
```

Comment:
If I'm reading this right, this is inside both the `for-i` and `for-j` loops. `idealRemappingPartialAssemblyName` only seems to depend on `i`, but not `j`.

I think maybe it's better to keep the bool `foundAtLeastOneUnresolvableConflict`, and set it to true as before, but log the warning outside of `for-j` but inside the `for-i` loop, so that it is output only once per each assembly name.

I didn't test my theory, it's all just from reading the code and psychic debugging. It'd be great to have unit-tests support or deny this.